tendrl/ceph_integration/manager/cluster_monitor.py

```diff
@@ -350,14 +350,23 @@ def on_sync_object(self, data):
                 ) else None,
                 now(), sync_object, self._manager.cluster_id)
             if sync_type.str == "osd_map":
+                util_data = self._get_utilization_data()
                 for raw_pool in sync_object.get('pools', []):
                     LOG.info("Updating Pool %s" % raw_pool['pool_name'])
+                    for pool in util_data['pools']:
+                        if pool['name'] == raw_pool['pool_name']:
+                            max_avail = pool['stats']['max_avail']
+                            pool_used = pool['stats']['bytes_used']
+                            pcnt = (pool_used * 100) / (pool_used + max_avail)
                     pool = Pool(updated=str(time.time()),
                                 cluster_id=self._manager.cluster_id,
                                 pool_id=raw_pool['pool'],
                                 poolname=raw_pool['pool_name'],
                                 pg_num=raw_pool['pg_num'],
-                                min_size=raw_pool['min_size']
+                                min_size=raw_pool['min_size'],
+                                used=pool_used,
+                                total=max_avail,
```

Comment:
max_avail is not total we might need to remove this and have either only percent_used or percent_used and used